frontend/src/components/TogglePage.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import ToggleButton from "react-bootstrap/ToggleButton";
+
+function ToggleButtonExample(props) {
+
+    const [radioValue, setRadioValue] = useState(props.props.countPerPage);
+
+    const radios = [
+        {name: '10', value: '10'},
+        {name: '20', value: '20'},
+    ];
+
+
+    return (
+        <>
+            <ButtonGroup toggle>
+                {radios.map((radio, idx) => (
```

Comment:
Please, move this map processing outside the render function, create a variable, and use here just the result.